galleries/joseph/style.css

```
@@ -0,0 +1,47 @@
+* {
+    padding: 0;
+    margin: 0;
+}
+
+header {
+    background-color: darkcyan;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 1rem;
+}
+
+main {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+}
+
+.gallery_images {
+    border: 2px solid red;
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
```

```suggestion
    grid-template-columns: repeat(3, 1fr);
```
Since this is what the wireframe wants.